src/UrlGeneratorInterface.php

```diff
@@ -45,13 +45,11 @@ public function generateAbsolute(
         string $host = null
     ): string;
 
+    public function generateCurrent(array $replacedParams, string $fallbackRouteName = null): string;
+
     public function getUriPrefix(): string;
 
     public function setUriPrefix(string $name): void;
 
-    public function getLocales(): array;
-
-    public function setLocales(array $locales): void;
-
-    public function setLocaleParameterName(string $localeParameterName): void;
+    public function setDefault(string $name, $value): void;
```

Comment:
Is there case for non-string value?
```suggestion
    public function setDefault(string $name, string $value): void;
```

Reply:
That's done for convenience not to cast to string every time.

Reply:
This can be not a string. This method is not only for locales.